packages/sanity/src/core/form/inputs/DateInputs/DateTimeInput.tsx

```diff
@@ -65,7 +80,8 @@ function enforceTimeStep(dateString: string, timeStep: number) {
     return serialize(setMinutes(date, minutes - leftOver))
   }
 
-  return serialize(date)
+  const serilalised = serialize(date)
+  return serilalised
```

Comment:
Bit of a mismatch between US and UK spelling here, could we just return the value of `serialize`?

```suggestion
  return serialize(date)
```